configserv/image.yaml

```diff
@@ -1,4 +1,4 @@
-version: 1.0
+version: latest
```

Comment:
The `latest` tag you get always in the build system (as well, as others). This is the place where you actually specify the version of the image.